### ISSUE TITLE
aws_lambda_function : tracing_config always triggers changes in regions not supporting X-Ray for lambda

### ISSUE THREAD

**Issue by ebarault** [terraform-providers/terraform-provider-aws#3058]
Terraform v0.11.2

I noticed that when deploying lambda functions in regions that do not support X-Ray for Lambda, such as Paris eu-west-3, Terraform always triggers changes on  `tracing_config.#`.

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ aws_lambda_function.lambda
      tracing_config.#: "" => <computed>

Plan: 0 to add, 1 to change, 0 to destroy.
```

My lambda function definition does not include the  `tracing_config` block.

**Comment by joelittlejohn** [comment]
Seeing this problem in cn-north-1 too.

**Comment by wyattwalter** [comment]
We had this issue as well. Upgrading the AWS provider (currently `1.32.0`) addressed it for us.

**Comment by bflad** [comment]
Thanks for reporting this, @wyattwalter! Briefly looking at the handling for `tracing_config` I see a relevant change was made in #4668, which might have fixed this issue. That change was previously released in [version 1.21.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1210-may-31-2018) and has been available in all releases since. I'm guessing that closing this issue was just missed during merging of that pull request.

If anyone still has a related issue to this on the latest versions, please feel free to open a new issue. Thanks!
